sotodlib/tod_ops/fft_ops.py

```diff
@@ -1,17 +1,32 @@
 """FFTs and related operations
 """
 from scipy.signal import welch
+from scipy.fft import set_global_backend
 import numpy as np
 import pyfftw
+import logging
 
 import so3g
 
 from . import detrend_data
 
+logger = logging.getLogger(__name__)
+
+
 def _get_num_threads():
     # Guess how many threads we should be using in FFT ops...
     return so3g.useful_info().get('omp_num_threads', 4)
 
+
+# Setup scipy operations to use fftw
+try:
+    pyfftw.config.NUM_THREADS = _get_num_threads()
+    set_global_backend(pyfftw.interfaces.scipy_fft)
+    pyfftw.interfaces.cache.enable()
+except ValueError:
+    logger.debug("Unable to set pyffw backend")
```

Comment:
Please change the message to "Failed to configure pyfftw as scipy fft backend" (fix spelling, mention scipy).